mta/tests/migration/controls/test_business_services.py

```diff
@@ -0,0 +1,25 @@
+import pytest
+from pytest_check import check
+
+from conftest import generate_description, generate_name
+
+
+@pytest.mark.business
```

Comment:
1. Please change the marker to `bs`
2. Any new custom marker should be added to the `pytest.ini` file. For more info see: https://docs.pytest.org/en/7.1.x/example/markers.html#registering-markers